README.md

```diff
@@ -41,3 +41,20 @@ Install snowdrop in your npm project:
 $ npm install @uoguelph/snowdrop
 ```
 
+To reference a specific Github branch, you can use the following syntax in your package.json file:
+```
+  "@uoguelph/snowdrop": "github:ccswbs/snowdrop#feature-branch",
+```
+
+## Updating files
+
+1. Update the files under the scss directory
+2. If you wish, you can run `npm run dist` to update the distribution files.
```

Comment:
Now that I see it in print, `npm run dist' should really be run before every commit. I know when you are just casually editing a branch it doesn't really matter, but as far as the readme goes it would be better to update dist every time so we get in the habit (or just automate it.) The second item could just read "Run `npm run dist' to update the distribution files.'